virttest/libvirt_xml/devices/shmem.py

```diff
@@ -0,0 +1,57 @@
+"""
+shared memory device support class(es)
+
+<devices>
+  <shmem name='my_shmem0' role='peer'>
+    <model type='ivshmem-plain'/>
```

Comment:
`<address type="pci" ...>` is optional for shmem device, too. See the example of XML: https://gitlab.com/libvirt/libvirt/-/blob/master/tests/qemuxml2argvdata/shmem-plain-doorbell.xml#L32
@chunfuwen Please add that in the code comment and `__init__()`

Reply:
@qiankehan 
updated as per comments

Reply:
@qiankehan
updated as per comments

Reply:
@qiankehan
updated as per comments

Reply:
LGTM